shared/router-v2/switcheroo.js

```diff
@@ -28,4 +39,13 @@ class RouterSwitcheroo extends PureComponent<Props> {
   }
 }
 
-export default RouterSwitcheroo
+const mapDispatchToProps = dispatch => ({
+  persistRoute: path => dispatch(ConfigGen.createPersistRoute({path})),
+  updateNavigator: navigator => dispatch(ConfigGen.createSetNavigator({navigator})),
```

Comment:
It would be nice if we didn't have this

Reply:
Which aspect? We need a way to bridge redux and this. This is what they suggest right?

Reply:
They suggest this? where? I think it would be good to move away from dispatching anything other than pure data (Immutable stuff and HiddenString is fine since I count it as data).

That said, this in particular seems easy to fix in the future, so I wouldn't block on this. Just wanted to point out my thoughts.